packages/notebook-extension/src/index.ts

```diff
@@ -1718,7 +1718,8 @@ function addCommands(
     isEnabled
   });
   commands.addCommand(CommandIDs.restartClear, {
-    label: trans.__('Restart Kernel and Clear All Outputs…'),
+    label: trans.__('Restart Kernel and Clear Outputs of All Cells…'),
```

Comment:
Was there a consensus on the recommendation from @krassowski about removing the ellipsis? Here is the original comment:

>I think that three dots make sense for "Change Kernel..." because it indicates that a pop-up dialog with more options will be shown (and for "Go to line..." and for "Find..." as all these require additional user action), but it indeed does not seem necessary here (or for any other "Restart Kernel" option).

Reply:
From the [Apple Human Interface Guidelines on Menu Anatomy](https://developer.apple.com/design/human-interface-guidelines/macos/menus/menu-anatomy/), "Use an ellipsis whenever choosing a menu item requires additional input from the user. The ellipsis character (…) means a dialog or separate window will open and prompt the user for additional information or to make a choice."

If the kernel is running, all "Restart Kernel" options will spawn a dialog box asking for confirmation (Restart) or to abort (Cancel). The dialog box also provides additional information, that all variables will be lost, that may help a user make a more informed decision.

The ellipsis also conveys to the user that it is safe for them to click the menu item even if they're not totally sure what it will do, as they should expect to see a dialog box appear. This encourages users to explore the UI and to discover what all the different menu items do.

Reply:
Sounds good. Thanks for explaining.